File: spring-web-java/src/main/java/spring/web/java/repository/custom/MemberCustomRepository.java
package spring.web.java.repository.custom;

import org.springframework.data.domain.Page;
import org.springframework.data.domain.Pageable;
import spring.web.java.domain.Member;

public interface MemberCustomRepository {

    Page<Member> findAllWithJpql(Pageable pageable, String account, String name);

    Page<Member> findAllWithQuerydsl(Pageable pageable, String account, String name);
}

File: spring-web-kotlin/src/main/kotlin/spring/web/kotlin/repository/custom/MemberCustomRepository.kt
package spring.web.kotlin.repository.custom

import org.springframework.data.domain.Page
import org.springframework.data.domain.Pageable
import spring.web.kotlin.domain.Member

interface MemberCustomRepository {

    fun findAllWithJpql(pageable: Pageable, account: String?, name: String?): Page<Member>
}